core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeCommandBuilder.scala

```diff
@@ -0,0 +1,55 @@
+package com.databricks.labs.remorph.parsers.snowflake
+
+import com.databricks.labs.remorph.parsers.snowflake.SnowflakeParser._
+import com.databricks.labs.remorph.parsers.{IncompleteParser, ParserCommon, intermediate => ir}
+
+class SnowflakeCommandBuilder
+    extends SnowflakeParserBaseVisitor[ir.Command]
+    with ParserCommon[ir.Command]
+    with IncompleteParser[ir.Command] {
+
+  private val expressionBuilder = new SnowflakeExpressionBuilder
+
+  protected override def wrapUnresolvedInput(unparsedInput: String): ir.UnresolvedCommand =
+    ir.UnresolvedCommand(unparsedInput)
+
+  // [TODO]: Implement Cursor and Exception for Declare Statements.
```

Comment:
```suggestion
  // TODO: Implement Cursor and Exception for Declare Statements.
```
let's start all todo's with `// TODO` consistently - it'll be easier to autocreate issues from those.